engine/edge.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Docker CE Edge releases
+description: Information and changelogs for Docker CE Edge releasers
+---
+
+## Overview
+
+Docker CE Edge releases are monthly releases cut straight from the `master`
```

Comment:
_every_ release (except patch releases) is cut from master, quarterly releases as well. "straight from the master" sounds as if the raw bytes from master were directly published, which is not correct; edge releases follow the same procedure as quarterly releases, and also get a release-candidate / feature freeze.